Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+"@context": {
+	"sample": "<http://example.org>",
+	"rdfs": "<http://www.w3.org/2000/01/rdf-schema#> ",
+	"properties": "http://example.org/Properties#>",
+	"xsd": "<http://www.w3.org/2001/XMLSchema#>"
+},
```

Comment:
(This comment applies to all this file. Please, validate your code before submitting it. There are some syntax errors. You might use the json-ld playground online tool to test it. )

Your prefixes are wrong, check how they should be defined. Also, check the “sample” URI, something is missing at the end.